bin/make-tag.py

```diff
@@ -81,7 +81,7 @@ def run():
     # Create tag
     input(">>> Ready to tag? Ctrl-c to cancel")
     print(">>> Creating tag...")
-    subprocess.check_call(["git", "tag", "-s", "-a", tag_name, "-m", message])
+    subprocess.check_call(["git", "tag", "-s", tag_name, "-m", message])
```

Comment:
`-s` makes a signed annotated tag whereas `-a` makes an *unsigned* annotated tag. I'm pretty sure the `-a` was canceling the `-s` out. Regardless, we don't want `-a`.